app/player/player.controller.js

```diff
@@ -3,12 +3,15 @@
 angular.module('app.player')
 
 .controller('playerCtrl', ['$localStorage', function($localStorage) {
+	var addButtonClicked = false;
```

Comment:
What's the scope of this button? You should add some comments or a docblock for each private and public property.